specification/schemas/locales/Locale.json

```diff
@@ -0,0 +1,26 @@
+{
+  "allOf": [
+    {
+      "$ref": "#/components/schemas/LocaleResource"
+    },
+    {
+      "properties": {
+        "attributes": {
+          "type": "object",
+          "properties": {
+            "code": {
+              "$ref": "#/components/schemas/LocaleEnum"
+            },
+            "name": {
+              "type": "string",
+              "example": "English"
+            },
+            "created_at": {
+              "$ref": "#/components/schemas/Timestamp"
+            }
```

Comment:
I'd remove that since I cannot think of a reason to have it exposed!

Reply:
removed